app_flutter/lib/service/dev_cocoon.dart

```diff
@@ -246,6 +246,9 @@ class DevelopmentCocoonService implements CocoonService {
       ..requiredCapabilities.add('[linux/android]')
       ..reservedForAgentId = 'linux1'
       ..stageName = stageName
-      ..status = status;
+      ..status = status
+      ..buildNumberList = '$index'
```

Comment:
This will update every task to have these values. Check if the the task will be a luci task before assigning these.

Reply:
Good point. Added luci check before assigning.